node/cn/api_tracer.go

```diff
@@ -755,15 +758,27 @@ func (api *PrivateDebugAPI) traceTx(ctx context.Context, message blockchain.Mess
 				return nil, err
 			}
 		}
-		// Constuct the JavaScript tracer to execute with
-		if tracer, err = tracers.New(*config.Tracer); err != nil {
-			return nil, err
+
+		if *config.Tracer == fastCallTracer {
+			tracer = vm.NewInternalTxTracer()
+		} else {
+			// Constuct the JavaScript tracer to execute with
+			if tracer, err = tracers.New(*config.Tracer); err != nil {
+				return nil, err
+			}
 		}
 		// Handle timeouts and RPC cancellations
 		deadlineCtx, cancel := context.WithTimeout(ctx, timeout)
 		go func() {
 			<-deadlineCtx.Done()
-			tracer.(*tracers.Tracer).Stop(errors.New("execution timeout"))
+			switch t := tracer.(type) {
```

Comment:
```suggestion
			tracer.Stop(errors.New("execution timeout"))
```
Does all tracers support `Stop()`?

Reply:
Unfortunately, no. Tracer interface doesn't have `Stop()` method.

Reply:
How about adding Stop to the Tracer interface?

Reply:
@ehnuje As you know, `Tracer` interface has only Capture series. For consistency, adding `Stop` method looks not good. Also, there is another tracer which does not have `Stop` although it can be newly added.

Reply:
How about converting it to `Stoppable` which has `Stop` method?

Reply:
If the function is not added to the `Tracer` interface, we need to cast the `Tracer` interface to the `Stoppable` interface, which is being done in this code. Let's think about this later if we have additional tracers in the future.